db/snapshot/backup_task.cc

```diff
@@ -175,27 +201,29 @@ future<> backup_task_impl::process_snapshot_dir() {
     }
 }
 
-future<> backup_task_impl::uploads_worker() {
-    named_gate uploads(format("do_backup::uploads({})", _snapshot_dir));
+future<> backup_task_impl::worker::uploads_worker() {
```

Comment:
Why not name this method `backup_task_impl::worker::start_uploading()`? The method of the same name exists, it's API one and it _just_ call this one anyway

Reply:
Or call `uploads_worker` above instead of `start_uploading`